openhtf/plugs/user_input.py

```diff
@@ -166,13 +158,12 @@ def respond(self, prompt_id, response):
 
 
 def prompt_for_test_start(
-    message='Provide a DUT ID in order to start the test.', text_input=True,
-    timeout_s=60*60*24):
-  """Make a test start trigger based on prompting the user for input."""
-  def trigger():  # pylint: disable=missing-docstring
-    plug = UserInput()
-    response = plug.prompt(
-        message, text_input=text_input, timeout_s=timeout_s)
-    plug.tearDown()
-    return response
-  return trigger
+    message='Enter a DUT ID in order to start the test.', timeout_s=None):
+  """Return an OpenHTF phase for use as a prompt-based start trigger."""
+  @plugs.plug(prompts=UserInput)
+  def trigger_phase(test, prompts):
+    """Test start trigger that prompts the user for a DUT ID."""
+    test.test_record.dut_id = prompts.prompt(message=message, text_input=True,
```

Comment:
Actually, this could become a single phase that `prompt_for_test_start()` just returns `WithArgs()` of, for backwards compatibility